homeassistant/components/vizio/strings.json

```diff
@@ -4,19 +4,35 @@
         "step": {
             "user": {
                 "title": "Setup Vizio SmartCast Device",
+                "description": "All fields are required except Access Token. If you choose not to provide an Access Token, and your Device Type is 'tv', you will go through a pairing process with your device so an Access Token can be retrieved.\n\nTo go through the pairing process, before clicking Submit, ensure your TV is powered on and connected to the network. You also need to be able to see the screen.",
                 "data": {
                     "name": "Name",
                     "host": "<Host/IP>:<Port>",
                     "device_class": "Device Type",
                     "access_token": "Access Token"
                 }
+            },
+            "pair_tv": {
+                "title": "Complete Pairing Process",
+                "description": "Your TV should be displaying a code. Enter that code into the form and then continue to the next step to complete the pairing.",
+                "data": {
+                    "pin": "PIN"
+                }
+            },
+            "pairing_complete": {
+                "title": "Pairing Complete",
+                "description": "Your Vizio SmartCast device is now connected to Home Assistant.\n\nYour Access Token is '**{access_token_value}**'."
+            },
+            "pairing_complete_import": {
+                "title": "Pairing Complete",
+                "description": "Your Vizio SmartCast device is now connected to Home Assistant.\n\nYour Access Token is '**{access_token_value}**'.\n\nThe access token has been stored with your configuration, but it is recommended that you add the token to your config so that you don't have to re-pair your device in the future."
```

Comment:
Why recommend to add access token to your configuration.yaml ?

Reply:
See comment above: https://github.com/home-assistant/home-assistant/pull/31520#discussion_r383669607

This only shows up when the source was import (e.g. if a user sets up the integration completely through the UI, configuration.yaml will not be mentioned)

Reply:
After re-reading, I agree that this is wordy and is unnecessarily taking the place of documentation. I kept the access token in the message but removed the recommendation which can be covered in the docs